src/Web/E2E/playwright.config.ts

```diff
@@ -29,10 +28,10 @@ const config: PlaywrightTestConfig = {
     /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
     actionTimeout: 0,
     /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: process.env.BASEURL,
+    baseURL: process.env.BASEURL || 'https://dotnetpodcasts.azurewebsites.net',
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: 'on',
-    video: 'on',
+    video: 'off',
```

Comment:
videos should not be used, since they are heavy cpu and memory wise. Using Playwright Tracing is the recommended approach.